cli_composer.go

```diff
@@ -143,7 +144,12 @@ func noninteractiveComposeMessage(subject string, attachments []string,
 	}
 
 	msg := fbb.NewMessage(fbb.Private, fOptions.MyCall)
-	msg.SetFrom(fOptions.MyCall)
+	if len(from) > 0 {
+		msg.SetFrom(from)
+
```

Comment:
I think we should refer to this as "sender address/callsign" to highlight that this could be used to set an email address as the sender as well as a callsign.

Also, the correct term is "tactical addresses" IIRC.

Reply:
Sorry abou that. This comment is on the wrong line. Should be seen with regards to the usage text.

Reply:
Done (though I wouldn't want to encourage a non-WL email address for the reasons you mentioned previously)

I went with "Address to send from. Default is your call from config or --mycall, but can be specified to use tactical addresses."